app/lib/pantry_trak/client.rb

```diff
@@ -69,5 +71,15 @@ def create_reservation_path
         'api/create_freshtrak_reservation_beta.php'
       end
     end
+
+    def handle_unsucessful_response(response)
+      # TODO: In order to rescue the very specific exceptions,
+      # refs# https://github.com/lostisland/faraday/blob/master/lib/faraday/error.rb
+      # Trying to handle exception and print on logs
+      Jets.logger.info '_' * 10
```

Comment:
Do we really need these '_______' lines? I see them as cruft (especially since they're at a different log level than the actual message).